src/__experimental__/components/decimal/decimal.component.js

```diff
@@ -256,10 +256,13 @@ class Decimal extends React.Component {
    * Format a user defined value
    */
   formatValue = (value) => {
-    invariant(
-      !this.isNaN(value),
-      `The supplied decimal (${value}) is not a number`
```

Comment:
Removing this means the component will error with `Cannot read property 'visibleValue' of undefined`.

Previously it would have errored with `The supplied decimal (12a.05) is not a number`.

```js
<Decimal label="Decimal" defaultValue="12a.05" />
```

Reply:
Messaged you privately about this comment 😃

Reply:
Reinstated the invariant as per the screenshot. 

<img width="1420" alt="Screenshot 2021-02-04 at 10 50 34" src="https://user-images.githubusercontent.com/56251247/106882565-d0f9d300-66d6-11eb-93f0-e8b22422931b.png">